tests/kafkatest/services/monitor/jmx.py

```diff
@@ -25,7 +27,7 @@ class JmxMixin(object):
     - we assume the service using JmxMixin also uses KafkaPathResolverMixin
     - this uses the --wait option for JmxTool, so the list of object names must be explicit; no patterns are permitted
     """
-    def __init__(self, num_nodes, jmx_object_names=None, jmx_attributes=None):
+    def __init__(self, num_nodes, jmx_object_names=None, jmx_attributes=None, root="/mnt"):
```

Comment:
I agree that we should move away from any service dumping directly into /mnt eventually.  However, at the moment, there are still services in muckrake that use the JMX mixin.  We don't want to break those with this change.  Keeping the JMX default at /mnt for now makes this change non-breaking.